flake.nix

```diff
@@ -48,7 +56,7 @@
         inherit (nixpkgs) lib;
 
         # see flake `variants` below for alternative compilers
-        defaultCompiler = "ghc966";
+        defaultCompiler = "ghc965";
```

Comment:
Why is the downgrade?

Reply:
There is not later ghc in the haskell.nix pin :(